soc/nordic/common/nrf_sys_event.c

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <nrf_sys_event.h>
+
+#if CONFIG_SOC_SERIES_NRF54HX
+
+/*
+ * The 54HX is not yet supported by an nrfx driver nor the system controller so
```

Comment:
Is there a PR to nrfx to add support for 54H?

Reply:
no, I have heard it will exist at some point, no more than that at the time

Reply:
Please create a PR to nrfx, this way using HAL will be hidden under abstraction.
Until we have nrfs part, which in my understanding will require separate ref counting, whole file can be simplified to just calling `nrfx_power`.

Reply:
its not that simple :) The H20 has no power peripheral, hence no `nrfx_power` driver. The handling of latency in H20 is completely novel, and performed through LRCs.

Reply:
that's why the comment says "nrfx driver", since I for one have no idea how it can be abstracted by nrfx

Reply:
nrfx drivers do not necessarily use that single peripheral. If the functionality is equal, we try to abstract that underneath. See for example here: https://github.com/nrfconnect/nrfx/blob/master/drivers/src/nrfx_power.c#L92-L96

But let's discuss offline later, not a blocker from my side.